ui/app/src/Utils/String.elm

```diff
@@ -0,0 +1,14 @@
+module Utils.String exposing (capitalizeFirst)
+
+import String exposing (left, length, slice, cons, uncons, trim)
```

Comment:
If you're using `String.uncons` etc.,  then there is no need to import it in the scope.